docs/_static/custom.css

```diff
@@ -2,3 +2,19 @@
 body{font-family:'Roboto',sans-serif;}.wy-menu-vertical p.caption{color:#7cccc7;}
 *{font-variant-ligatures: none;}.autoclasstoc td {padding:0.2rem;line-height:normal;}
 dl.field-list>dt{word-break: normal}
+
+
+/* Instead of showing the [source] link for the github link, show a github icon */
```

Comment:
Looks like the [source] link is more common used in other repo, such as PyTorch? What do you think?

cc @ericspod @Nic-Ma

Reply:
I can find some examples of sphinx.ext.linkcode later

Reply:
I prefer linking to the file in the repo, it's cleaner and lets one browse to other places quickly given how Github is adding more dynamic features to code browsing. I've often found readthedocs and other pages which never link to the project's repo anywhere which I find really annoying when I just want to get to the code so this is helpful for that.

Reply:
As a general comment: The line of code or rather the whole change in custom.css is just so that we get one link to github using an icon and one linking to the current code copy in the docs using the usual [source] link. If we wouldn't have this change in custom.css, there would be [source][source] in this location.